notify/notify.yaml

```diff
@@ -7,8 +7,8 @@ servers:
 paths:
   /team-webhooks:
     get:
-      summary: Get all webhooks
-      description: This endpoint allows you to get all webhooks from every app on your team.
+      summary: Get all webhooks on your team
```

Comment:
no need to update the summary here. Adding the clarification in the description is sufficient